### PR TITLE
fix aeraki memory leak when node info update

### PR DESCRIPTION
**Pull request by shencurl521**
### Pre-Submission Checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added related test cases?
* [x] Have you modified the related document?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: internal/controller/istio/controller.go [hunk 1]
```diff
@@ -84,10 +84,6 @@ func (c *Controller) Run(stop <-chan struct{}) {
 	go c.configCache.Run(stop)
 	go func() {
 		c.connectIstio()
-		for {
```
[the remark this answers is not in the document]
Comment (shencurl521) [reply internal/controller/istio/controller.go:87]: ok i have had sign off

File: internal/controller/istio/controller.go [hunk 2]
```diff
@@ -125,6 +121,8 @@ func (c *Controller) connectIstio() {
 			sm, err := c.newSecretManager()
 			if err != nil {
 				controllerLog.Errorf("failed to create SecretManager %s %v", c.options.IstiodAddr, err)
+				time.Sleep(5 * time.Second)
+				continue
```
Comment (zhaohuabing) [inline internal/controller/istio/controller.go:125]: Could you please fix the go lint issue?

superfluous-else: if block ends with a continue statement, so drop this else and outdent its block (revive)

Comment (shencurl521) [reply internal/controller/istio/controller.go:125]: ok has fixed

**Review by zhaohuabing — APPROVED** [review]
LGTM. Thanks for fixing it.

**Pull request merged**